tests/config/test_config.py

```diff
@@ -0,0 +1,44 @@
+import tempfile
+
+import pytest
+
+from sflvault.config.config import Config
+
+# Create config file
+tf = tempfile.NamedTemporaryFile(dir='tests/data')
```

Comment:
Maybe we should put these in a fixture function (with module-wide scope)?

https://docs.pytest.org/en/latest/fixture.html